src/makeRobot.js

```diff
@@ -38,7 +38,84 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const Robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goRight(move) {
+      if (move < 0) {
+        return this;
+      } else if (move === undefined) {
+        this.coords.x += 1;
+
+        return this;
+      } else {
+        this.coords.x += move;
+
+        return this;
+      }
+    },
```

Comment:
You can simplify this code using default parameter value like:
```suggestion
    goRight(value = 1) {
      if (value > 0) {
        this.coords.x += value;
      };

      return this;
    },
```

You can apply this to other methods